app/views/pages/union.html.erb

```diff
@@ -0,0 +1,73 @@
+<% content_for :template_name, 'homepage' %>
+<% content_for :banner do
+  render "shared/banner", banner_text: "Welcome to MDHHS on Union Street!"
+end %>
+
+<%= render 'shared/flashes' %>
+
+<div class="slab--hero-<%= rand(1..3) %> slab slab--hero">
+  <div class="grid">
+    <div class="logo-hero">MichiganBenefits.org</div>
+    <h1>
+      Get the support your family needs
+    </h1>
+    <p>
+      Apply for food stamps (FAP) in 15 minutes
+    </p>
+    <%= link_to step_path(StepNavigation.first, office_location: "union"), class: "button button--cta button--fully-width" do %>
+      Apply now from MDHHS at Union St.
+      <i class="button__icon icon-arrow_forward"></i>
+    <% end %>
+  </div>
+</div>
+
+<!-- <div class="slab slab--white" id="apply-for-programs">
+  <div class="card--narrow">
+    <%= render 'shared/flashes' %>
+    <label class="program-selector">
+      <div class="program-selector__checkbox">
+        <% icon_class = "illustration--icn_food" %>
+      </div>
+      <div class="<%= icon_class %> program-selector__icon illustration">
+        <h4 class="program-selector__title">
+          Food Assistance
+        </h4>
+      </div>
+      <div class="program-selector__subtitle text--smell">
+        Food Stamps, FAP or SNAP
+      </div>
+      <div class="text--help">
+        Provides benefits to buy or grow food.
+      </div>
+    </label>
+  </div>
+</div> -->
+
+<section class="slab" id="scroller-target">
+  <div class="grid">
+    <div class="grid__item width-seven-twelfths">
+      <p>
+        MichiganBenefits.org is an easy, safe and secure way to apply for
+        Michigan's Food Assistance Program. Information you provide will be
+        securely submitted as an official application with the Michigan
+        Department of Health and Human Services.
+
+        MichiganBenefits.org is a non-profit that offers online application
+        assistance to any Michigan resident. For additional assistance email our
+        team at <a href="mailto:hello@michiganbenefits.org">hello@michiganbenefits.org</a>.
+      </p>
+
+      <p>
+        To visit the official online application for food assistance, please visit
+        <%= link_to 'MI Bridges', 'https://www.michigan.gov/mibridges' %>.
+      </p>
+    </div>
+
+    <div class="grid__item width-one-fourth shift-one-sixth trust-badge">
+      <a href="http://www.michigan.gov/mdhhs/" class="illustration illustration--mi_mdhhs">MDHHS</a>
+      <p class="text--small">A registered Community Partner and digital application assister.</p>
+    </div>
+  </div>
+</section>
+
+<%= render 'shared/footer' %>
```

Comment:
__maybe blocking__ - Since this is the same as index, with only super minor differences, does it make sense to merge them somehow?

Reply:
I am hesitant to remove all duplication because I think that makes these view templates harder to read and update, but I extracted the code that is exactly the same into a shared partial here: d2aa436